app/components/endpoint/endpoint.html

```diff
@@ -31,6 +31,17 @@
             </div>
           </div>
           <!-- !endpoint-url-input -->
+          <!-- endpoint-url-publish-input -->
+          <div class="form-group">
+            <label for="endpoint_url_publish" class="col-sm-3 col-lg-2 control-label text-left">
+              Public IP
+              <portainer-tooltip position="bottom" message="URL or IP address where the services will be exposed."></portainer-tooltip>
```

Comment:
Replace tooltip message with `URL or IP address where exposed containers will be reachable. This field is optional and will default to the endpoint URL.`